crate/pages/install/cloud/azure.html

```diff
@@ -0,0 +1,117 @@
+title: Crate & Microsoft Azure
+description:
+logo: /static/images/azure-logo.png
+
+{% extends "install.html" %}
+
+{% block install_content %}
+
+<div class="row">
+  <div class="col-sm-8">
+    <p>
+      Microsoft Azure is the second largest and fastest growing prvidor of Cloud Services in the World. It offers a wide variety of options including Windows servers, Containers, Application images and much more.
+    </p>
+    <p>
+      Getting Crate working on Azure with Linux or Windows is a simple process. You can use Azure's management console or CLI interface (<a href="https://azure.microsoft.com/en-us/documentation/articles/xplat-cli-install/" target="_blank">Learn how to install here</a>).
+      <h2>Azure & Linux</h2>
+      <h3>Create a Virtual Network</h3>
+      <p>
+        To create a cluster of Crate nodes on cloud hosting we need to rely on unicast for inter-node communcation, as multicast will not be available.
+      </p>
+      <p>
+        The easiest way to get Unicast communication working with Azure is to create a Virtual Network (<i>Network Services -> Virtual Network -> Custom Create</i>) so that all of our nodes exist on the same IP range. Give the network a name, a region and let Azure handle all the remaining settings by clicking the next arrow on each screen.
+      </p>
+      <p>
+        <img class="img-responsive" src="/static/images/azure-vpc.png" />
+      </p>
+      <h3>Create Virtual Machines</h3>
+      <p>
+        Next create virtual machines to act as your Crate nodes. In this tutorial, I chose two low-specification Ubuntu 14.04 servers, but you likely have your own preferred configurations. The important step is number 4, where we need to select the Virtual Network created earlier.
+      </p>
+      <p>
+        <img class="img-responsive" src="/static/images/azure-vn-select.png" />
+      </p>
+      <p>
+        Also at stage 4 we can create the endpoints required for opening the ports Crate needs for inter-node communication, 4200 and 4300. You should end up with something like:
```

Comment:
inter-node communication is on port `4300` (transport port with custom protocol), the java client is also using that port
`4200` is the HTTP port, which needs to be open for all other crate clients